### PR TITLE
dev-cmd/contributions: Count PR reviews since they're super important

### PR DESCRIPTION
**Pull request by issyl0**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew typecheck` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----

- Part of https://github.com/Homebrew/brew/issues/13642.
- The search APIs don't have that high a rate limit but we shouldn't need to worry about that too much because, to get counts, the JSON response comes with a `total_count` number.

### CODE REVIEW

**Review by nandahkrishna — APPROVED** [review]
Thanks, @issyl0!

**Review by nandahkrishna — APPROVED** [review]

File: Library/Homebrew/dev-cmd/contributions.rb
Comment (MikeMcQuaid) [inline Library/Homebrew/dev-cmd/contributions.rb:178]: Definitely wouldn't have been blocking: is there any way to query the specific type of reviews?

Comment (issyl0) [reply Library/Homebrew/dev-cmd/contributions.rb:178]: A query like `is:pr reviewed-by:#{user} review:approved` works to get approvals by the user, but we can't combine `review:approved` and `review:changes_requested`. So I thought that just `reviewed-by:#{user}` was safest to encompass everything.

Comment (MikeMcQuaid) [reply Library/Homebrew/dev-cmd/contributions.rb:178]: @issyl0 I'd consider filtering this to just `review:approved` as it'll get fewer results, requires write access (i.e. "maintainer duties") and is required to move things forward. Thoughts?

**Pull request merged**
